src/components/SideBar/SideBar.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { SideBar } from './SideBar';
+
+export default {
+  title: 'Components/Side Bar',
+  component: SideBar,
+} as ComponentMeta<typeof SideBar>;
+
+const Template: ComponentStory<typeof SideBar> = (args) => (
+  <SideBar {...args} />
+);
+
+export const MainSideBar = Template.bind({});
```

Comment:
What is the difference between main sidebar and side bar? A comment or two would help